unittests/kv_tests.cpp

```diff
@@ -40,7 +40,6 @@ FC_REFLECT(itparam, (db)(count)(erase))
 class kv_tester : public tester {
  public:
```

Comment:
Should all the references to DISK in these tests be removed?

Reply:
As the test contracts are still in the old format, those references to DISK cannot be removed for now.
We are doing the removal in phases. They will be removed in next phase when focusing on tests,
or removed in the final phase when CDT removes DISK.